src/category_theory/sites/dense_subsite.lean

```diff
@@ -19,12 +19,12 @@ we would need, and some sheafification would be needed for here and there.
 ## Main results
 
 - `category_theory.cover_dense.sheaf_hom`: If `G : C ⥤ (D, K)` is full and cover-dense,
-then given any sheaves `ℱ, ℱ'` on `D`, and a morphism `α : G ⋙ ℱ ⟶ G ⋙ ℱ'`, we may glue them
-together to obtain a morphsim of sheaves `ℱ ⟶ ℱ'`.
+  then given any presheaf `ℱ` and sheaf `ℱ'` on `D`, and a morphism `α : G ⋙ ℱ ⟶ G ⋙ ℱ'`,
+  we may glue them together to obtain a morphsim of sheaves `ℱ ⟶ ℱ'`.
```

Comment:
Looks great! The typo `morphsim` and `isomorphsim` [below](https://github.com/leanprover-community/mathlib/pull/9694/commits/71bfcd5aab0a2451902033b4045251d2e2945b4c#diff-4a971c16350870036568147a6226f23328a30c23e6a46a94279ca00224fefe35R242) remain uncorrected, though.